gremlin-javascript/src/main/javascript/gremlin-javascript/lib/utils.js

```diff
@@ -24,6 +24,8 @@
 'use strict';
 
 const crypto = require('crypto');
+const os = require('os');
+const gremlinVersion = require(__dirname + '/../package.json').version;
```

Comment:
Let's confirm this still works even when the target client application is compiled in minimized js package (so reference to JSON file is not lost).

Reply:
I tried using webpack to package and minimize a test client app and the driver version was still being correctly loaded into the user agent without issue.